test/boost/memtable_test.cc

```diff
@@ -516,6 +518,51 @@ SEASTAR_TEST_CASE(test_exception_safety_of_single_partition_reads) {
     });
 }
 
+SEASTAR_TEST_CASE(test_memtable_reversing_reader) {
+    return seastar::async([] {
+        auto s = schema_builder("ks", "cf")
```

Comment:
tip: there is `simple_schema` which simplifies tests like this. No need to change already existing code though.

Reply:
I think `random_schema` will be a better fit, we want to make sure composite keys work too. I will soon publish v4 of my PR which will have an example test using `random_schema`.

Reply:
The exploratory testing should be covered by existing mutation source tests, we just need to run them with reverse readers.